lib/scala/pkg/src/main/java/org/enso/pkg/archive/EnsoProjectArchiveException.java

```diff
@@ -0,0 +1,11 @@
+package org.enso.pkg.archive;
+
+import java.io.IOException;
+
+/** Base class for errors when building enso-project archive. */
+abstract class EnsoProjectArchiveException extends IOException {
+
+  public EnsoProjectArchiveException(IOException cause) {
```

Comment:
Why do we need to wrap an `IOException` with a subclass of `IOException`? Can't we just propagate `IOException` directly? Just like `EnsoProjectArchive build(...) throws IOException` does?

Reply:
I wanted to distinguish those from other IO exceptions

Reply:
Distinguishing is OK. I also  created something like `EnsoProjectArchiveException(String msg)` exception that extends `IOException`. I just don't understand why it takes `IOException` as a delegate... but now I think that's OK.